### DISCUSSION TITLE
Setting the neutral-point of a gauge

### DISCUSSION THREAD

**Discussion by sfranzis** [grafana/grafana#38273] [Dashboards, Panels & Visualizations]
**What would you like to be added**:
The neutral or zero point of a gauge is at the left of the panel. When I set the range from e.g. Min=-5 to Max = 5 then the display of negative values looks not natural. 
I would like the neutral-point to be configurable in the standard field options (where we have Min, Max, ...).

**Why is this needed**:
My use case is a solar battery inverter, which shows positive values when charging and negative values when discharging.

This is how the gauge panel works now:
![gauge-1](https://user-images.githubusercontent.com/12957935/117032673-f613d700-ad01-11eb-85e5-8cc0846a309e.png)

This is what I would like to see - Neutral value = 0  (mockup not proportional to values):
![gauge-2](https://user-images.githubusercontent.com/12957935/117032700-fad88b00-ad01-11eb-89ea-2f325ba9ec82.png)

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by sfranzis** [comment]
Yesterday I pulled the repository and started implementing this.
Idea: set min and max to opposite values (e.g. -2500 and 2500) then draw the angles in a way, that zero is visibly at 12 o'clock.
This needs only a small change in jquery.flot.gauge.js and no new toggle in the panel options.

What do you think?

Image: state of my solar battery inverter discharging.
<img width="225" alt="battery inverter discharging" src="https://user-images.githubusercontent.com/12957935/184527506-2ccdac8f-8fc4-43d4-9508-9b645c2fc54a.png">

**Reply by sfranzis** [reply]
Pull request issued:
https://github.com/grafana/grafana/pull/53989

**Comment by sfranzis** [comment]
Feature implemented with Grafana 9.3.0 - https://github.com/grafana/grafana/blob/main/CHANGELOG.md
See: Gauge options => Neutral

Have fun,
Stefan

**Reply by pkkrusty** [reply]
Any interest in proposing/submitting a similar option for the retro lcd bar gauge? Where only a single bar is illuminated over the range of values? Or illuminating from a neutral point? Use case is solar inverter feed-in/export.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
